configs/_base_/datasets/vaihingen.py

```diff
@@ -0,0 +1,49 @@
+# dataset settings
+dataset_type = 'ISPRSDataset'
```

Comment:
The dataset type is not correct.

Reply:
The Potsdam and Vaihingen use the same dataset